ansible/provision_vps.yml

```diff
@@ -10,13 +10,17 @@
     docker_image_backend: "ghcr.io/kryptokrona/kryptokrona-api"
     docker_image_monitor: "ghcr.io/kryptokrona/kryptokrona-api-monitor"
 
+    # Docker hub Container Registry
+    docker_image_node: "jrkalf/kryptokrona-node"
```

Comment:
This will not work. There is no docker image on github registry by this name. I think. it was with his previous account. Maybe we can use our docker image instead for now. Here we have it: https://github.com/kryptokrona/kryptokrona/pkgs/container/kryptokrona

So it would be `kryptokrona/kryptokrona` instead. I have updated to expose 11897 port as well for the latest one.

Reply:
That will works since the image is on docker hub. And by default it will pull from there :)

Reply:
Ah ok! Great! Handy!